website/content/workbooks/workbook-10.md

```diff
@@ -0,0 +1,22 @@
++++
+title="CYF+02 Sprint 4"
+date="01 Jan 2024"    
+versions=["1-1-0"]
+weight=5
++++
+
+# Provisional start date: 20 May 2024
+
+## Study
+
+- [ ] [Distributed Systems: Asynchronous Work and Pipelines](../../primers/distributed-software-systems-architecture/asynchronous-work-and-pipelines)
+- [ ] [Troubleshooting Primer](../../primers/troubleshooting/)
+
+## Projects
+
+- [ ] [Kafka Cron](https://github.com/CodeYourFuture/immersive-go-course/tree/main/kafka-cron) - Distributed Cron system with Prometheus monitoring
+- [ ] [Troubleshooting project #4](https://docs.google.com/document/d/1V6HEu_OcJ3MHH-aHzUfANf06VJa1rPcGHcpBwql7QLA/edit#heading=h.cjnguaxmynan) TOADD
+
+## Product
+
+Monitoring and alerting is a key part of any production system. You will be working with your product team to understand the monitoring and alerting in place on your codebase.
```

Comment:
```suggestion
Monitoring and alerting is a key part of any production system. As you continue your product work, make sure to understand the monitoring and alerting in place on your codebase and how it could be improved.
```